tt_eager/tt_dnn/op_library/untilize/kernels/dataflow/halo_gather.cpp

```diff
@@ -102,7 +102,7 @@ void kernel_main() {
     constexpr uint32_t elem_nbytes = sizeof(uint16_t);
     constexpr uint16_t pad_core_id = 0xFFFF;
 
-    static_assert(stick_nbytes <= NOC_MAX_BURST_SIZE);  // stick_nbytes used in noc_async_read_one_packet
+    // static_assert(stick_nbytes <= NOC_MAX_BURST_SIZE);  // stick_nbytes used in noc_async_read_one_packet
```

Comment:
why disable this? i think this is needed.

Reply:
We were hitting the NOC_MAX_BURST_SIZE limit(8K). My stick size is [1, 5120]. With bfloat16 its 5120*2=10240 bytes.

Reply:
Isn't it safe to remove those asserts in halo_gather.cpp  as there are only `noc_async_read()` calls ? @tt-aho can you confirm?

Reply:
Yes there are no one packet api calls in this file, only `noc_async_read` which doesn't have a size limitation so it is safe to remove.